get_number.rb

```diff
@@ -0,0 +1,48 @@
+# Get My Number Game
+# Written by: you!
+
+puts "Welcome to 'Get My Number!'"
```

Comment:
Have you ever checked the code with rubocop?

Reply:
ok